autosubmitconfigparser/config/configcommon.py

```diff
@@ -588,12 +588,13 @@ def _normalize_dependencies(dependencies: Union[str, dict]) -> dict:
 
     @staticmethod
     def _normalize_files(files: str) -> List[str]:
```

Comment:
This seems ok to me. I'll just add this typing.

```suggestion
    def _normalize_files(files: Union[str, List[str]]) -> List[str]:
```

Reply:
Thanks! changed